gradient/api_sdk/clients/secret_client.py

```diff
@@ -0,0 +1,61 @@
+from .base_client import BaseClient
+from .. import models, repositories
+from ...exceptions import ReceivingDataFailedError
+
+SECRET_ENTITIES = ("cluster", "project", "team")
+
+
+class SecretsClient(BaseClient):
+    entity = "secret"
```

Comment:
This is not required here, I think. This is used for tags. Unless one can add tags to secrets?

Reply:
Nope, will remove.